File: practices/fizzbuzz.py
# SC 7th Fizzbuzz

# GO BACK TO NOTES

x = 1

while x < 51:
    if x % 3 == 0 and x % 5 == 0:
        print("FizzBuzz!")
    elif x % 5 == 0:
        print("Buzz!")
    elif x % 3 == 0: 
        print("Fizz!")
    else:
        print(x)
    x += 1

File: c_programming/practices/fizzbuzz.c
// SC 7th FizzBuzz
#include <stdio.h>

int main(void){

    for (int x = 1; x <= 50; x++){
    
        if (x % 3 == 0 && x % 5 == 0){
            printf("FizzBuzz!\n", x);
        }else if (x % 5 == 0){
            printf("Buzz!\n", x);
        }else if (x % 3 == 0){
            printf("Fizz!\n", x);
        }else{
            printf("%d\n", x);
        }
    }
       
    return 0;
}